lldb/include/lldb/API/SBTarget.h

```diff
@@ -964,6 +964,7 @@ class LLDB_API SBTarget {
   friend class SBSection;
   friend class SBSourceManager;
   friend class SBSymbol;
+  friend class SBType;
```

Comment:
These were only necessary to get access to `SBTarget::GetSP`

Reply:
Okay, sorry, this is my bad. I didn't mean to imply that it should be done in this patch. I do like this idea, but I think that if we do that, it should be done in a more wider fashion rather than a one-off.

Reply:
Ah fair enough, reverted that for now